src/app/(simulation)/(large-layout)/amis/resultats/_components/groupResults/Classement.tsx

```diff
@@ -55,12 +63,13 @@ export default function Classement({ group }: { group: Group }) {
           }
 
           const { formattedValue, unit } = formatCarbonFootprint(
-            participant?.simulation?.computedResults[defaultMetric].bilan ?? ''
+            participant?.simulation?.computedResults?.[defaultMetric]?.bilan ??
```

Comment:
Why adding this optional marks (more below)

Reply:
Removed it, no lint errors